apps/webapp/src/fetchers/address.ts

```diff
@@ -31,19 +24,16 @@ export const getAddresses = async (accounts: (number | undefined)[]): Promise<In
     }, {});
 };
 
-export const getAddressByIndex = async (account: number | undefined): Promise<Address> => {
-  const req = new AddressByIndexRequest();
-  if (account) req.addressIndex = new AddressIndex({ account });
-  const res = await viewClient.addressByIndex(req);
-  if (!res.address) throw new Error('Address not in getAddressByIndex response');
-  return res.address;
+export const getAddressByIndex = async (account = 0): Promise<Address> => {
```

Comment:
I go back and forth between whether we should have default account numbers for these functions. If someone is trying to get an address, they never should be passing nothing right?

Reply:
well, *we* are sometimes passing nothing, from our own frontend. so

Reply:
imo passing nothing should be interpreted as the default account (index 0)